Tests/PLCrashAsyncMachOImageTests.m

```diff
@@ -48,8 +48,9 @@ @implementation PLCrashAsyncMachOImageTests
 - (void) setUp {
     /* Fetch our containing image's dyld info */
     Dl_info info;
-    STAssertTrue(dladdr([self class], &info) > 0, @"Could not fetch dyld info for %p", [self class]);
-
+    void *classRef = (__bridge void *)([self class]);
+    STAssertTrue(dladdr(classRef, &info) > 0, @"Could not fetch dyld info for %p", [self class]);
+    CFBridgingRelease(classRef);
```

Comment:
__bridge doesn't incur a retain, and the `-class` reference is a borrowed one; it should not be explicitly released.